### ISSUE TITLE
Another Burp Import issue after custom pip tld install - "from tld import get_tld" 

### ISSUE THREAD

**Issue by Os3r3um** [pathetiq/BurpSmartBuster#12]
Hi dude, 

Firstly, thanks for your work. Secondly I do have a problem installing this one unfortuantelly. 
So, I got followed your steps and everything decribed bellow was recorded with a Kali r2 2016, fully updated. 

That being said here's what's happening trying to make your extension working( beautifulsoup and requests were already installed with my system)

Accordign with your wiki:
"
Prerequisite:

Install Jython 2.7
Inside Jython (using pip from jython bin folder) install:

git clone https://github.com/sloria/TextBlob.git
git branch tag/0.11.1
[apt-get install python-beautifulsoup]
[apt-get install python-request]
pip install tld
"

My steps: 
1. Got installed jython2.7 from official jar installation file. 
2. root@kali2016r2:~/jython2.7.0# git clone https://github.com/sloria/TextBlob.git
   Cloning into 'TextBlob'...
   remote: Counting objects: 3392, done.
   remote: Total 3392 (delta 0), reused 0 (delta 0), pack-reused 3392
   Receiving objects: 100% (3392/3392), 7.87 MiB | 288.00 KiB/s, done.
   Resolving deltas: 100% (1829/1829), done.
   Checking connectivity... done.

root@kali2016r2:~/jython2.7.0# cd TextBlob/
root@kali2016r2:~/jython2.7.0/TextBlob# git branch tag/0.11.1

root@kali2016r2:~/jython2.7.0/bin# ./pip install tld
Downloading/unpacking tld
  Downloading tld-0.7.6-py2.py3-none-any.whl (147kB): 147kB downloaded
Downloading/unpacking six>=1.4 (from tld)
  Downloading six-1.10.0-py2.py3-none-any.whl
Installing collected packages: tld, six
Successfully installed tld six
Cleaning up...

The py file was modified accordingly: 

sys.path.append("/usr/local/lib/python2.7/site-packages")
sys.path.append("/usr/lib/python2.7/dist-packages/nltk")
sys.path.append("~/jython2.7.0/TextBlob")
1. I fired up burp and when I'm trying to lauch the extension this is the error which I'm getting. 

Traceback (most recent call last):
  File "/root/Desktop/burp_suite/BurpSmartBuster/BurpSmartBuster.py", line 44, in <module>
    from tld import get_tld
ImportError: No module named tld

```
at org.python.core.Py.ImportError(Py.java:328)
at org.python.core.imp.import_first(imp.java:877)
at org.python.core.imp.import_module_level(imp.java:972)
at org.python.core.imp.importName(imp.java:1062)
at org.python.core.ImportFunction.__call__(__builtin__.java:1280)
at org.python.core.PyObject.__call__(PyObject.java:431)
at org.python.core.__builtin__.__import__(__builtin__.java:1232)
at org.python.core.imp.importFromAs(imp.java:1156)
at org.python.core.imp.importFrom(imp.java:1132)
at org.python.pycode._pyx1.f$0(/root/Desktop/burp_suite/BurpSmartBuster/BurpSmartBuster.py:1574)
at org.python.pycode._pyx1.call_function(/root/Desktop/burp_suite/BurpSmartBuster/BurpSmartBuster.py)
at org.python.core.PyTableCode.call(PyTableCode.java:167)
at org.python.core.PyCode.call(PyCode.java:18)
at org.python.core.Py.runCode(Py.java:1386)
at org.python.core.__builtin__.execfile_flags(__builtin__.java:535)
at org.python.util.PythonInterpreter.execfile(PythonInterpreter.java:286)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at burp.wee.<init>(Unknown Source)
at burp.aq.a(Unknown Source)
at burp.qwf.run(Unknown Source)
at java.lang.Thread.run(Thread.java:745)
```

Any ideas?

Thanks!

**Comment by pathetiq** [comment]
Hi, 

Thanks I appreciate it!

I'm pretty sure the problem is that there's a wrong path to site-packages. As an example mine is:  /home/username/Documents/Apps/jython27/Lib/site-packages/tld and "/usr/local/lib/python2.7/site-packages on another computer"

Make sure to find the directory with those files: 
 tld-0.7.6.dist-info
 tld
six.py

When this is done you can try "import tld" inside your jython console and see if that work.

Let me know if that works and if not just update the error you got.

Thanks

Patrick

**Comment by Os3r3um** [comment]
So, yeah. Dug into this a little deeper 'cause this is getting way too intense :)

There are couple of things which don't quite match with the initial wiki install procedure so I'll give you an update about what've did:

1. Got a fresh install of jython under a path like /tmp/jython2.7/ and then I went to the bin directory where I ran the following:

./pip beautifulsoup4 requests textblog

Everything went okay and packages were installed under /tmp/jython2.7/site-packages

This directory has these contents at the moment: 

beautifulsoup4-4.5.1.dist-info  
requests
bs4                             
requests-2.11.1.dist-info
colorama-0.3.7-py2.7.egg        
setuptools
easy-install.pth                
setuptools-14.3.2.dev0.dist-info
easy_install.py                 
six-1.10.0.dist-info
easy_install$py.class          
six.py
_markerlib                      s
ix$py.class
nltk                            
termcolor-1.1.0-py2.7.egg
nltk-3.2.1-py2.7.egg-info       
textblob
pip                             
textblob-0.11.1.dist-info
pip-1.6.dev1.dist-info          
tld
pkg_resources                  
tld-0.7.6.dist-info
README                          
tldr-0.3-py2.7.egg

I changed that path thing with the original python file: 

sys.path.append("/tmp/jython2.7.0/Lib/site-packages")
sys.path.append("/tmp/jython2.7.0/Lib/site-packages/nltk")
sys.path.append("/tmp/jython2.7.0/Lib/site-packages/textblob")

However, loading it through latest BurpPro I'm getting now this ugly error message which is something lese that I got intothe first place:

"
	at org.python.core.imp.createFromPyClass(imp.java:236)
	at org.python.core.imp.createFromPyClass(imp.java:205)
	at org.python.core.imp.loadFromSource(imp.java:651)
	at org.python.core.imp.find_module(imp.java:543)
	at org.python.core.PyModule.impAttr(PyModule.java:106)
	at org.python.core.imp.import_next(imp.java:842)
	at org.python.core.imp.import_logic(imp.java:904)
	at org.python.core.imp.import_module_level(imp.java:978)
	at org.python.core.imp.importName(imp.java:1062)
	at org.python.core.ImportFunction.__call__(__builtin__.java:1280)
	at org.python.core.PyObject.__call__(PyObject.java:431)
	at org.python.core.__builtin__.__import__(__builtin__.java:1232)
	at org.python.core.imp.importAll(imp.java:1182)
	at nltk.corpus.reader$py.f$0(/tmp/jython2.7.0/Lib/site-packages/nltk/corpus/reader/__init__.py:111)
	at nltk.corpus.reader$py.call_function(/tmp/jython2.7.0/Lib/site-packages/nltk/corpus/reader/__init__.py)
	at org.python.core.PyTableCode.call(PyTableCode.java:167)
	at org.python.core.PyCode.call(PyCode.java:18)
	at org.python.core.imp.createFromCode(imp.java:436)
	at org.python.core.imp.createFromPyClass(imp.java:236)
	at org.python.core.imp.createFromPyClass(imp.java:205)
	at org.python.core.imp.loadFromSource(imp.java:651)
	at org.python.core.imp.find_module(imp.java:543)
	at org.python.core.PyModule.impAttr(PyModule.java:106)
	at org.python.core.imp.import_next(imp.java:842)
	at org.python.core.imp.import_logic(imp.java:904)
	at org.python.core.imp.import_module_level(imp.java:978)
	at org.python.core.imp.importName(imp.java:1062)
	at org.python.core.ImportFunction.__call__(__builtin__.java:1280)
	at org.python.core.PyObject.__call__(PyObject.java:431)
	at org.python.core.__builtin__.__import__(__builtin__.java:1232)
	at org.python.core.imp.importAll(imp.java:1182)
	at nltk.corpus$py.f$0(/tmp/jython2.7.0/Lib/site-packages/nltk/corpus/__init__.py:315)
	at nltk.corpus$py.call_function(/tmp/jython2.7.0/Lib/site-packages/nltk/corpus/__init__.py)
	at org.python.core.PyTableCode.call(PyTableCode.java:167)
	at org.python.core.PyCode.call(PyCode.java:18)
	at org.python.core.imp.createFromCode(imp.java:436)
	at org.python.core.imp.createFromPyClass(imp.java:236)
	at org.python.core.imp.createFromPyClass(imp.java:205)
	at org.python.core.imp.loadFromSource(imp.java:651)
	at org.python.core.imp.find_module(imp.java:543)
	at org.python.core.PyModule.impAttr(PyModule.java:106)
	at org.python.core.imp.import_next(imp.java:842)
	at org.python.core.imp.import_logic(imp.java:904)
	at org.python.core.imp.import_module_level(imp.java:978)
	at org.python.core.imp.importName(imp.java:1062)
	at org.python.core.ImportFunction.__call__(__builtin__.java:1280)
	at org.python.core.PyObject.__call__(PyObject.java:431)
	at org.python.core.__builtin__.__import__(__builtin__.java:1232)
	at org.python.core.imp.importFromAs(imp.java:1156)
	at org.python.core.imp.importFrom(imp.java:1132)
	at nltk.stem.snowball$py.f$0(/tmp/jython2.7.0/Lib/site-packages/nltk/stem/snowball.py:3642)
	at nltk.stem.snowball$py.call_function(/tmp/jython2.7.0/Lib/site-packages/nltk/stem/snowball.py)
	at org.python.core.PyTableCode.call(PyTableCode.java:167)
	at org.python.core.PyCode.call(PyCode.java:18)
	at org.python.core.imp.createFromCode(imp.java:436)
	at org.python.core.imp.createFromPyClass(imp.java:236)
	at org.python.core.imp.createFromPyClass(imp.java:205)
	at org.python.core.imp.loadFromSource(imp.java:651)
	at org.python.core.imp.find_module(imp.java:543)
	at org.python.core.PyModule.impAttr(PyModule.java:106)
	at org.python.core.imp.import_next(imp.java:842)
	at org.python.core.imp.import_logic(imp.java:904)
	at org.python.core.imp.import_module_level(imp.java:978)
	at org.python.core.imp.importName(imp.java:1062)
	at org.python.core.ImportFunction.__call__(__builtin__.java:1280)
	at org.python.core.PyObject.__call__(PyObject.java:431)
	at org.python.core.__builtin__.__import__(__builtin__.java:1232)
	at org.python.core.imp.importFromAs(imp.java:1156)
	at org.python.core.imp.importFrom(imp.java:1132)
	at nltk.stem$py.f$0(/tmp/jython2.7.0/Lib/site-packages/nltk/stem/__init__.py:31)
	at nltk.stem$py.call_function(/tmp/jython2.7.0/Lib/site-packages/nltk/stem/__init__.py)
	at org.python.core.PyTableCode.call(PyTableCode.java:167)
	at org.python.core.PyCode.call(PyCode.java:18)
	at org.python.core.imp.createFromCode(imp.java:436)
	at org.python.core.imp.createFromPyClass(imp.java:236)
	at org.python.core.imp.createFromPyClass(imp.java:205)
	at org.python.core.imp.loadFromSource(imp.java:651)
	at org.python.core.imp.find_module(imp.java:543)
	at org.python.core.PyModule.impAttr(PyModule.java:106)
	at org.python.core.imp.import_next(imp.java:842)
	at org.python.core.imp.import_logic(imp.java:904)
	at org.python.core.imp.import_module_level(imp.java:978)
	at org.python.core.imp.importName(imp.java:1062)
	at org.python.core.ImportFunction.__call__(__builtin__.java:1280)
	at org.python.core.PyObject.__call__(PyObject.java:431)
	at org.python.core.__builtin__.__import__(__builtin__.java:1232)
	at org.python.core.imp.importAll(imp.java:1182)
	at nltk$py.f$0(/tmp/jython2.7.0/Lib/site-packages/nltk/__init__.py:184)
	at nltk$py.call_function(/tmp/jython2.7.0/Lib/site-packages/nltk/__init__.py)
	at org.python.core.PyTableCode.call(PyTableCode.java:167)
	at org.python.core.PyCode.call(PyCode.java:18)
	at org.python.core.imp.createFromCode(imp.java:436)
	at org.python.core.imp.createFromPyClass(imp.java:236)
	at org.python.core.imp.createFromPyClass(imp.java:205)
	at org.python.core.imp.loadFromSource(imp.java:651)
	at org.python.core.imp.find_module(imp.java:543)
	at org.python.core.imp.import_next(imp.java:840)
	at org.python.core.imp.import_module_level(imp.java:959)
	at org.python.core.imp.importName(imp.java:1062)
	at org.python.core.ImportFunction.__call__(__builtin__.java:1280)
	at org.python.core.PyObject.__call__(PyObject.java:431)
	at org.python.core.__builtin__.__import__(__builtin__.java:1232)
	at org.python.core.imp.importOne(imp.java:1081)
	at textblob.blob$py.f$0(/tmp/pip_build_root/textblob/textblob/blob.py:690)
	at textblob.blob$py.call_function(/tmp/pip_build_root/textblob/textblob/blob.py)
	at org.python.core.PyTableCode.call(PyTableCode.java:167)
	at org.python.core.PyCode.call(PyCode.java:18)
	at org.python.core.imp.createFromCode(imp.java:436)
	at org.python.core.imp.createFromPyClass(imp.java:236)
	at org.python.core.imp.createFromPyClass(imp.java:205)
	at org.python.core.imp.loadFromSource(imp.java:651)
	at org.python.core.imp.find_module(imp.java:543)
	at org.python.core.PyModule.impAttr(PyModule.java:106)
	at org.python.core.imp.import_next(imp.java:842)
	at org.python.core.imp.import_module_level(imp.java:959)
	at org.python.core.imp.importName(imp.java:1062)
	at org.python.core.ImportFunction.__call__(__builtin__.java:1280)
	at org.python.core.PyObject.__call__(PyObject.java:431)
	at org.python.core.__builtin__.__import__(__builtin__.java:1232)
	at org.python.core.imp.importFromAs(imp.java:1156)
	at org.python.core.imp.importFrom(imp.java:1132)
	at textblob$py.f$0(/tmp/pip_build_root/textblob/textblob/__init__.py:11)
	at textblob$py.call_function(/tmp/pip_build_root/textblob/textblob/__init__.py)
	at org.python.core.PyTableCode.call(PyTableCode.java:167)
	at org.python.core.PyCode.call(PyCode.java:18)
	at org.python.core.imp.createFromCode(imp.java:436)
	at org.python.core.imp.createFromPyClass(imp.java:236)
	at org.python.core.imp.createFromPyClass(imp.java:205)
	at org.python.core.imp.loadFromSource(imp.java:651)
	at org.python.core.imp.find_module(imp.java:543)
	at org.python.core.imp.import_next(imp.java:840)
	at org.python.core.imp.import_module_level(imp.java:959)
	at org.python.core.imp.importName(imp.java:1062)
	at org.python.core.ImportFunction.__call__(__builtin__.java:1280)
	at org.python.core.PyObject.__call__(PyObject.java:431)
	at org.python.core.__builtin__.__import__(__builtin__.java:1232)
	at org.python.core.imp.importFromAs(imp.java:1156)
	at org.python.core.imp.importFrom(imp.java:1132)
	at org.python.pycode._pyx1.f$0(/root/Desktop/burp_suite/BurpSmartBuster/BurpSmartBuster.py:1580)
	at org.python.pycode._pyx1.call_function(/root/Desktop/burp_suite/BurpSmartBuster/BurpSmartBuster.py)
	at org.python.core.PyTableCode.call(PyTableCode.java:167)
	at org.python.core.PyCode.call(PyCode.java:18)
	at org.python.core.Py.runCode(Py.java:1386)
	at org.python.core.__builtin__.execfile_flags(__builtin__.java:535)
	at org.python.util.PythonInterpreter.execfile(PythonInterpreter.java:286)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at burp.wee.<init>(Unknown Source)
	at burp.aq.a(Unknown Source)
	at burp.qwf.run(Unknown Source)
	at java.lang.Thread.run(Thread.java:745)
"

**Comment by pathetiq** [comment]
dumb question. did you update the jython.jar path in burp?

**Comment by pathetiq** [comment]
Also make sure to point to the installation of beautifulsoup and request. something like : /usr/local/lib/python2.7/dist-packages/

**Comment by Os3r3um** [comment]
So, I will follow all installation process once again to be sure that I'm not missing anything. This plugin is way to good not be used :)

**Comment by pathetiq** [comment]
Alright thanks. Let me know what happen and I'll do it on my side two if that fails. I have the exact same setup as you so it's suppose to be straightforward.

**Comment by Os3r3um** [comment]
Hi dude, I'm still working on this but from some reasons my spare time is limited atm. Will come back you next week with an update. Cheers.

**Comment by pathetiq** [comment]
Thanks for the update. I'll also try to reinstall it over the weekend.

**Comment by pathetiq** [comment]
Please see this step of installation: https://github.com/pathetiq/BurpSmartBuster/issues/13#issuecomment-254255508 and see if that works for you

**Comment by Os3r3um** [comment]
Hi dude, 

Got a look today on this matter and I tried that installation steps. Burp Pro v1.7.09.

"
my installation steps OS kali linux :
java -jar jython_installer-2.7.0.jar
cd /root/jython2.7.0/bin
./pip install beautifulsoup4 requests tld textblob
changes -> BurpSmartBuster.py :

sys.path.append("/root/jython2.7.0/Lib/site-packages/")
sys.path.append("/root/jython2.7.0/Lib/site-packages/textblob")
sys.path.append("/root/jython2.7.0/Lib/site-packages/nltk")

BurpSuite :
Tab Extender - Tab Options - Python Environment : path /root/jython2.7.0/jpython.jar
"

Not working for me though. I'm getting the following error message:

"
	at org.python.core.imp.createFromCode(imp.java:436)
	at org.python.core.imp.createFromPyClass(imp.java:236)
	at org.python.core.imp.createFromPyClass(imp.java:205)
	at org.python.core.imp.loadFromSource(imp.java:651)
	at org.python.core.imp.find_module(imp.java:543)
	at org.python.core.PyModule.impAttr(PyModule.java:106)
	at org.python.core.imp.import_next(imp.java:842)
	at org.python.core.imp.import_logic(imp.java:904)
	at org.python.core.imp.import_module_level(imp.java:978)
	at org.python.core.imp.importName(imp.java:1062)
	at org.python.core.ImportFunction.__call__(__builtin__.java:1280)
	at org.python.core.PyObject.__call__(PyObject.java:431)
	at org.python.core.__builtin__.__import__(__builtin__.java:1232)
	at org.python.core.imp.importAll(imp.java:1182)
	at nltk.corpus.reader$py.f$0(/root/jython2.7.0/Lib/site-packages/nltk/corpus/reader/__init__.py:111)
	at nltk.corpus.reader$py.call_function(/root/jython2.7.0/Lib/site-packages/nltk/corpus/reader/__init__.py)
	at org.python.core.PyTableCode.call(PyTableCode.java:167)
	at org.python.core.PyCode.call(PyCode.java:18)
	at org.python.core.imp.createFromCode(imp.java:436)
	at org.python.core.imp.createFromPyClass(imp.java:236)
	at org.python.core.imp.createFromPyClass(imp.java:205)
	at org.python.core.imp.loadFromSource(imp.java:651)
	at org.python.core.imp.find_module(imp.java:543)
	at org.python.core.PyModule.impAttr(PyModule.java:106)
	at org.python.core.imp.import_next(imp.java:842)
	at org.python.core.imp.import_logic(imp.java:904)
	at org.python.core.imp.import_module_level(imp.java:978)
	at org.python.core.imp.importName(imp.java:1062)
	at org.python.core.ImportFunction.__call__(__builtin__.java:1280)
	at org.python.core.PyObject.__call__(PyObject.java:431)
	at org.python.core.__builtin__.__import__(__builtin__.java:1232)
	at org.python.core.imp.importAll(imp.java:1182)
	at nltk.corpus$py.f$0(/root/jython2.7.0/Lib/site-packages/nltk/corpus/__init__.py:315)
	at nltk.corpus$py.call_function(/root/jython2.7.0/Lib/site-packages/nltk/corpus/__init__.py)
	at org.python.core.PyTableCode.call(PyTableCode.java:167)
	at org.python.core.PyCode.call(PyCode.java:18)
	at org.python.core.imp.createFromCode(imp.java:436)
	at org.python.core.imp.createFromPyClass(imp.java:236)
	at org.python.core.imp.createFromPyClass(imp.java:205)
	at org.python.core.imp.loadFromSource(imp.java:651)
	at org.python.core.imp.find_module(imp.java:543)
	at org.python.core.PyModule.impAttr(PyModule.java:106)
	at org.python.core.imp.import_next(imp.java:842)
	at org.python.core.imp.import_logic(imp.java:904)
	at org.python.core.imp.import_module_level(imp.java:978)
	at org.python.core.imp.importName(imp.java:1062)
	at org.python.core.ImportFunction.__call__(__builtin__.java:1280)
	at org.python.core.PyObject.__call__(PyObject.java:431)
	at org.python.core.__builtin__.__import__(__builtin__.java:1232)
	at org.python.core.imp.importFromAs(imp.java:1156)
	at org.python.core.imp.importFrom(imp.java:1132)
	at nltk.stem.snowball$py.f$0(/root/jython2.7.0/Lib/site-packages/nltk/stem/snowball.py:3642)
	at nltk.stem.snowball$py.call_function(/root/jython2.7.0/Lib/site-packages/nltk/stem/snowball.py)
	at org.python.core.PyTableCode.call(PyTableCode.java:167)
	at org.python.core.PyCode.call(PyCode.java:18)
	at org.python.core.imp.createFromCode(imp.java:436)
	at org.python.core.imp.createFromPyClass(imp.java:236)
	at org.python.core.imp.createFromPyClass(imp.java:205)
	at org.python.core.imp.loadFromSource(imp.java:651)
	at org.python.core.imp.find_module(imp.java:543)
	at org.python.core.PyModule.impAttr(PyModule.java:106)
	at org.python.core.imp.import_next(imp.java:842)
	at org.python.core.imp.import_logic(imp.java:904)
	at org.python.core.imp.import_module_level(imp.java:978)
	at org.python.core.imp.importName(imp.java:1062)
	at org.python.core.ImportFunction.__call__(__builtin__.java:1280)
	at org.python.core.PyObject.__call__(PyObject.java:431)
	at org.python.core.__builtin__.__import__(__builtin__.java:1232)
	at org.python.core.imp.importFromAs(imp.java:1156)
	at org.python.core.imp.importFrom(imp.java:1132)
	at nltk.stem$py.f$0(/root/jython2.7.0/Lib/site-packages/nltk/stem/__init__.py:31)
	at nltk.stem$py.call_function(/root/jython2.7.0/Lib/site-packages/nltk/stem/__init__.py)
	at org.python.core.PyTableCode.call(PyTableCode.java:167)
	at org.python.core.PyCode.call(PyCode.java:18)
	at org.python.core.imp.createFromCode(imp.java:436)
	at org.python.core.imp.createFromPyClass(imp.java:236)
	at org.python.core.imp.createFromPyClass(imp.java:205)
	at org.python.core.imp.loadFromSource(imp.java:651)
	at org.python.core.imp.find_module(imp.java:543)
	at org.python.core.PyModule.impAttr(PyModule.java:106)
	at org.python.core.imp.import_next(imp.java:842)
	at org.python.core.imp.import_logic(imp.java:904)
	at org.python.core.imp.import_module_level(imp.java:978)
	at org.python.core.imp.importName(imp.java:1062)
	at org.python.core.ImportFunction.__call__(__builtin__.java:1280)
	at org.python.core.PyObject.__call__(PyObject.java:431)
	at org.python.core.__builtin__.__import__(__builtin__.java:1232)
	at org.python.core.imp.importAll(imp.java:1182)
	at nltk$py.f$0(/root/jython2.7.0/Lib/site-packages/nltk/__init__.py:184)
	at nltk$py.call_function(/root/jython2.7.0/Lib/site-packages/nltk/__init__.py)
	at org.python.core.PyTableCode.call(PyTableCode.java:167)
	at org.python.core.PyCode.call(PyCode.java:18)
	at org.python.core.imp.createFromCode(imp.java:436)
	at org.python.core.imp.createFromPyClass(imp.java:236)
	at org.python.core.imp.createFromPyClass(imp.java:205)
	at org.python.core.imp.loadFromSource(imp.java:651)
	at org.python.core.imp.find_module(imp.java:543)
	at org.python.core.imp.import_next(imp.java:840)
	at org.python.core.imp.import_module_level(imp.java:959)
	at org.python.core.imp.importName(imp.java:1062)
	at org.python.core.ImportFunction.__call__(__builtin__.java:1280)
	at org.python.core.PyObject.__call__(PyObject.java:431)
	at org.python.core.__builtin__.__import__(__builtin__.java:1232)
	at org.python.core.imp.importOne(imp.java:1081)
	at textblob.blob$py.f$0(/tmp/pip_build_root/textblob/textblob/blob.py:690)
	at textblob.blob$py.call_function(/tmp/pip_build_root/textblob/textblob/blob.py)
	at org.python.core.PyTableCode.call(PyTableCode.java:167)
	at org.python.core.PyCode.call(PyCode.java:18)
	at org.python.core.imp.createFromCode(imp.java:436)
	at org.python.core.imp.createFromPyClass(imp.java:236)
	at org.python.core.imp.createFromPyClass(imp.java:205)
	at org.python.core.imp.loadFromSource(imp.java:651)
	at org.python.core.imp.find_module(imp.java:543)
	at org.python.core.PyModule.impAttr(PyModule.java:106)
	at org.python.core.imp.import_next(imp.java:842)
	at org.python.core.imp.import_module_level(imp.java:959)
	at org.python.core.imp.importName(imp.java:1062)
	at org.python.core.ImportFunction.__call__(__builtin__.java:1280)
	at org.python.core.PyObject.__call__(PyObject.java:431)
	at org.python.core.__builtin__.__import__(__builtin__.java:1232)
	at org.python.core.imp.importFromAs(imp.java:1156)
	at org.python.core.imp.importFrom(imp.java:1132)
	at textblob$py.f$0(/tmp/pip_build_root/textblob/textblob/__init__.py:11)
	at textblob$py.call_function(/tmp/pip_build_root/textblob/textblob/__init__.py)
	at org.python.core.PyTableCode.call(PyTableCode.java:167)
	at org.python.core.PyCode.call(PyCode.java:18)
	at org.python.core.imp.createFromCode(imp.java:436)
	at org.python.core.imp.createFromPyClass(imp.java:236)
	at org.python.core.imp.createFromPyClass(imp.java:205)
	at org.python.core.imp.loadFromSource(imp.java:651)
	at org.python.core.imp.find_module(imp.java:543)
	at org.python.core.imp.import_next(imp.java:840)
	at org.python.core.imp.import_module_level(imp.java:959)
	at org.python.core.imp.importName(imp.java:1062)
	at org.python.core.ImportFunction.__call__(__builtin__.java:1280)
	at org.python.core.PyObject.__call__(PyObject.java:431)
	at org.python.core.__builtin__.__import__(__builtin__.java:1232)
	at org.python.core.imp.importFromAs(imp.java:1156)
	at org.python.core.imp.importFrom(imp.java:1132)
	at org.python.pycode._pyx1.f$0(/root/Desktop/burp_suite/mywebappscripts/BurpExtensions/BurpSmartBuster/BurpSmartBuster.py:1575)
	at org.python.pycode._pyx1.call_function(/root/Desktop/burp_suite/mywebappscripts/BurpExtensions/BurpSmartBuster/BurpSmartBuster.py)
	at org.python.core.PyTableCode.call(PyTableCode.java:167)
	at org.python.core.PyCode.call(PyCode.java:18)
	at org.python.core.Py.runCode(Py.java:1386)
	at org.python.core.__builtin__.execfile_flags(__builtin__.java:535)
	at org.python.util.PythonInterpreter.execfile(PythonInterpreter.java:286)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at burp.jw.<init>(Unknown Source)
	at burp.tig.a(Unknown Source)
	at burp.kme.run(Unknown Source)
	at java.lang.Thread.run(Thread.java:745)
"

**Comment by pathetiq** [comment]
I update the installation notes and added a fix in the code for the plugin to load correctly. Your issue is related to nltk and textblob version. Your error log isn't full because you copied it from burp output and note a file or the console that's why it was hard to debug.

https://github.com/pathetiq/BurpSmartBuster/commit/23f95bc18c038620e6604d6a7a519fbe4c939b29
https://github.com/pathetiq/BurpSmartBuster/wiki
